activerecord/test/cases/connection_adapters/connection_handlers_multi_db_test.rb

```diff
@@ -330,15 +330,22 @@ def test_retrieve_connection
       end
 
       def test_active_connections?
-        assert_not_predicate @handler, :active_connections?
+        ActiveSupport::Deprecation.silence do
```

Comment:
Should this use `assert_deprecated` instead?

Reply:
Fixed this. I dont know why I used silence at first.